UnityProject/Assets/StreamingAssets/TechWeb/Designs/MiscDesigns.json

```diff
@@ -139,6 +139,45 @@
         "name": "Welding Gas Mask",
         "Item_ID": "/obj/item/clothing/mask/gas/welding"
     },
+    {
+        "Internal_ID": "gas_mask",
```

Comment:
Try not to use different naming conventions for these. I saw a lot of them being alllowercase above. Follow that or change the rest to be snake_case

Reply:
have done snake case as its far more readable for ID names